homeassistant/helpers/llm.py

```diff
@@ -58,11 +58,6 @@
 
 LLM_API_ASSIST = "assist"
 
-BASE_PROMPT = (
-    'Current time is {{ now().strftime("%H:%M:%S") }}. '
-    'Today\'s date is {{ now().strftime("%Y-%m-%d") }}.\n'
```

Comment:
My assumption is that this was only used in chat_log.py but if that's not the case then I can bring it back